### PR TITLE
Add jason to feedstock maintainers

### PR DESCRIPTION
**Pull request by SylvainCorlay**
@jasongrout

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,9 @@
 # This file was generated automatically from conda-smithy. To update this configuration,
-# update the conda-forge.yml and/or the recipe/meta.yaml.
+# update the conda-forge.yaml and/or the recipe/meta.yaml.
 
 language: generic
 
 os: osx
-osx_image: beta-xcode6.1
```
Comment (jakirkham) [inline .travis.yml]: Hmm...what version of `conda-smithy` do you have? We want `1.4.6`.

Comment (SylvainCorlay) [inline .travis.yml]: Oh I did not update it correctly. Re-rerendered the feedstock.

**Comment by SylvainCorlay** [comment]
Same here. Travis build failing. I cancelled the rest of the build.

```
The command "MINICONDA_URL="http://repo.continuum.io/miniconda"
  MINICONDA_FILE="Miniconda3-latest-MacOSX-x86_64.sh"
  curl -O "${MINICONDA_URL}/${MINICONDA_FILE}"
  bash $MINICONDA_FILE -b
  
  export PATH=/Users/travis/miniconda3/bin:$PATH
  conda config --add channels conda-forge
  conda config --set show_channel_urls true
  conda install --yes --quiet conda-forge-build-setup
  source run_conda_forge_build_setup
  " failed and exited with 1 during .
Your build has been stopped.
/Users/travis/build.sh: line 138: shell_session_update: command not found
```

**Comment by jakirkham** [comment]
So I think your new re-rendering should fix it. AFAICT it looks right.

**Comment by SylvainCorlay** [comment]
Done in other PR.

**Pull request closed without merging**